Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG NODE_VERSION=14.20.1
+ARG NODE_VERSION=18.16.0
```

Comment:
Shouldn’t this be 16.20.0 too?

Reply:
It should but I think there is a general follow up to pull this correctly. I don't think we have verified if this Dockerfile works